src/fundus/publishers/uk/the_bbc.py

```diff
@@ -0,0 +1,53 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class TheBBCParser(ParserProxy):
+    class V1(BaseParser):
+        _subheadline_selector = CSSSelector("div[data-component='subheadline-block']")
+        _paragraph_selector = CSSSelector("div[data-component='text-block']")
+
+        @attribute
+        def body(self) -> ArticleBody:
+            return extract_article_body_with_selector(
+                self.precomputed.doc,
+                summary_selector=self._subheadline_selector,
```

Comment:
You accidentally added your subheadline selector as your summary selector

Reply:
Woops, good spot, thanks!